addons/mail/static/src/models/composer/composer.js

```diff
@@ -188,6 +188,10 @@ function factory(dependencies) {
          * @returns {mail.partner[]}
          */
         _computeRecipients() {
+            if (this.thread && this.thread.model === 'mail.channel') {
+                // prevent from notifying/adding to followers non-members
+                return [['unlink-all']];
+            }
```

Comment:
Problem with this approach is that members will not receive the notification either, but since we removed sending members at init for channels, we're kinda stuck here...

Reply:
task-2392127